apps/pie-docs/src/components/tag/code/props.json

```diff
@@ -17,7 +17,7 @@
             "variant",
             {
                 "type": "code",
-                "item": ["\"neutral\"", "\"neutral-alternative\"", "\"outline\"", "\"ghost\"", "\"blue\"", "\"green\"", "\"yellow\"", "\"red\"", "\"brand\"", "\"brand-03\"", "\"brand-04\"", "\"brand-06\""]
+                "item": ["\"neutral\"", "\"neutral-alternative\"", "\"outline\"", "\"ghost\"", "\"information\"", "\"success\"", "\"error\"", "\"brand-02\"", "\"brand-03\"", "\"brand-04\"", "\"brand-05\"", "\"brand-06\""]
```

Comment:
Do we also need to update the docs page?
<img width="1325" alt="Screenshot 2024-11-08 at 14 45 40" src="https://github.com/user-attachments/assets/8a33674b-3833-4f9e-992e-f0fdf007b80f">

Reply:
this will be done in  DSW-2324